pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[project]
+name = 'neo3-boa'
+description = 'A Python compiler for the Neo3 Virtual Machine'
+readme = 'README.md'
+requires-python = '>= 3.10.0'
+license = {file = 'LICENSE' }
+keywords = ['compiler', 'NEO', '.nef', 'blockchain', 'smartcontract', 'development', 'dApp']
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Intended Audience :: Developers',
+    'Topic :: Software Development :: Build Tools',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python :: 3.10',
+]
+dynamic = ['version']
+dependencies = [
+    'base58>=1.0.3',
+    'wheel>=0.30.0',
+    'requests==2.31.0',
+]
+
+[project.optional-dependencies]
+dev = [
+    'autopep8>=1.4.4',
+    'base58>=1.0.3',
```

Comment:
`base58` and `wheel` are already present in the standard required dependencies. We should remove them here.